### DISCUSSION TITLE
MFA and Github Education

### DISCUSSION THREAD

**Discussion by sinclairjamie** [community/community#17710] [GitHub Education] [answered]
Wondering if anyone knows if this announcement ( https://github.blog/2022-05-04-software-security-starts-with-the-developer-securing-developer-accounts-with-2fa ) will impact Github Education accounts as well?

**Comment by pattacini** [comment, marked as answer]
Yes, as GitHub Education is simply built on top of GitHub.

When the Personal Access Tokens (PAT) were made mandatory on GitHub last summer, there were already some hot discussions on how this had impacted the students' workflow.

See for example https://education.github.community/t/new-personal-access-tokens/69562/15.
